src/components/LenderProfile/LenderProfileWrapper.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-	<div>
+	<kv-page-container
```

Comment:
Why did we need to move this container?

Reply:
In order to have the full width gray background for badges section.